platform/web/tests/decode_with_json_schema_test.go

```diff
@@ -40,58 +40,47 @@ var testCases = []decodeWithJSONSchemaTestData{
 		},
 	},
 	{
-		Name:           "when a parameter has an incorrect datatype",
+		Name:           "when_a_parameter_has_an_incorrect_datatype",
```

Comment:
Must the test names be snaked cased now ?

Reply:
I did it based on this answer: https://github.com/fewlinesco/catalog/pull/443#discussion_r621895160

Reply:
yep saw it a bit later :)

Reply:
Based on the answer, I thought the team already settled on what should be done. If not the case, we can bring the topic to Slack or ping the others.

Reply:
Itmight have happenned when I was not here

Reply:
https://fewlines.slack.com/archives/CBQTY8P0T/p1611736007091000

Reply:
well it's even a proposition of mine haha